tools/make/packaging.mk

```diff
@@ -374,6 +374,7 @@ ifeq ($(USE_CONTAINER),1)
 else
 	cp ./dist/grafana-agent-windows-amd64.exe ./packaging/grafana-agent/windows
 	cp LICENSE ./packaging/grafana-agent/windows
+	"mkdir" -p dist
```

Comment:
Does `mkdir` really have to be in quotes here? It's always a single word with no spaces so it shouldn't need it.

Ditto on line 386 below.

Reply:
Without quotes, there will be an failure under Windows.

It seems like the cmd.exe builtin mkdir command does not support the `-p` switch while the mkdir.exe (C:/Windows/system32/) support it.

The quotes forces to use the external exe instead the builtin command.

I found the solution here: https://superuser.com/questions/1253369/gnuwin32-makefile-mkdir-p

Reply:
That’s odd, and something we won’t remember :). Can you add a comment for each instance of this so we don’t accidentally remove the quotes in a future commit?